components/widgets/menu/menu_items/submenu_item.jsx

```diff
@@ -0,0 +1,182 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import * as Utils from 'utils/utils.jsx';
+import {ModalIdentifiers} from 'utils/constants';
+import SubMenuModal from '../menu_modals/submenu_modal/submenu_modal.jsx';
+import {openModal} from 'actions/views/modals';
+import store from 'stores/redux_store';
+
+// Requires an object conforming to a submenu structure passed to registerPostDropdownMenuAction
```

Comment:
Is this comment correct? Or did you mean structure passed to `registerPostDropdownSubMenuItem `

Reply:
updated the comment to be `registerPostDropdownSubMenuAction`